libraries/AP_KDECAN/AP_KDECAN_config.h

```diff
@@ -0,0 +1,9 @@
+
+#pragma once
+
+#include <AP_HAL/AP_HAL_Boards.h>
+
+#ifndef AP_KDECAN_ENABLED
+    #define AP_KDECAN_ENABLED (HAL_MAX_CAN_PROTOCOL_DRIVERS && !HAL_MINIMIZE_FEATURES && BOARD_FLASH_SIZE > 1024)
```

Comment:
```suggestion
    #define AP_KDECAN_ENABLED (HAL_MAX_CAN_PROTOCOL_DRIVERS && BOARD_FLASH_SIZE > 1024)
```

All minimized boards are 1MB

Reply:
done